roadmap/implementers-guide/src/node/backing/candidate-backing.md

```diff
@@ -27,7 +27,7 @@ The [Collator Protocol][CP] subsystem is the primary source of non-overseer mess
 
 This subsystem requests validation from the [Candidate Validation][CV] and generates an appropriate [`Statement`][Statement]. All `Statement`s are then passed on to the [Statement Distribution][SD] subsystem to be gossiped to peers. When [Candidate Validation][CV] decides that a candidate is invalid, and it was recommended to us to second by our own [Collator Protocol][CP] subsystem, a message is sent to the [Collator Protocol][CP] subsystem with the candidate's hash so that the collator which recommended it can be penalized.
 
-The subsystem should maintain a set of handles to Candidate Backing Jobs that are currently live, as well as the relay-parent to which they correspond.
+<!-- The subsystem should maintain a set of handles to Candidate Backing Jobs that are currently live, as well as the relay-parent to which they correspond. -->
```

Comment:
Good job! No need to leave comments though, you can remove the lines entirely. (This applies to the rest of the PR as well.)

Reply:
Ok will update my PR soon

Reply:
@mrcnski can you check one more time to see if changes done are correct or not?

Reply:
Sure, will look tomorrow. Thanks!